### ISSUE TITLE
Support using C-GET rather than C-MOVE for DICOM Retrieval

### ISSUE THREAD

**Issue by cancan101** [chop-dbhi/dicom-pipeline#10]
This would simplify the client by not requiring a PACS server to run locally.

**Comment by jeffmax** [comment]
I agree this is a nice idea as long as the server is setup to support C-GET, which I am not sure is always the case do to security issues.  We have had a re-occurring issue where the pipeline for some reason fails to stop the DICOM server and then the next run does not work properly because the old server is still active.
